fastai/dataset.py

```diff
@@ -269,8 +269,8 @@ class ImageClassifierData(ImageData):
     @property
     def is_multi(self): return self.trn_dl.dataset.is_multi
 
-    @classmethod
-    def get_ds(self, fn, trn, val, tfms, test=None, **kwargs):
+    @staticmethod
```

Comment:
You may be right in changing this to Static Method, but it might be better to leave it as classmethod since this is only called by other classmethods.